test/screenplay/pa_messages/pa_message_test.exs

```diff
@@ -27,6 +27,38 @@ defmodule Screenplay.PaMessages.PaMessageTest do
     }
   end
 
+  describe "get_all_messages/0" do
+    test "returns all messages with most recent first" do
+      insert(:pa_message, %{
+        id: 1,
+        start_time: ~U[2024-05-01T01:00:00Z],
+        end_time: ~U[2024-05-01T13:00:00Z],
+        days_of_week: [3]
+      })
+
+      Process.sleep(1000)
+
+      insert(:pa_message, %{
+        id: 2,
+        start_time: ~U[2024-05-02T12:00:00Z],
+        end_time: ~U[2024-05-02T12:00:00Z],
+        days_of_week: [3]
+      })
+
+      Process.sleep(1000)
+
+      insert(:pa_message, %{
+        id: 3,
+        start_time: ~U[2024-05-02T12:00:00Z],
+        end_time: ~U[2024-05-02T12:00:00Z],
+        days_of_week: [3]
+      })
+
+      assert [%PaMessage{id: 3}, %PaMessage{id: 2}, %PaMessage{id: 1}] =
+               PaMessage.get_all_messages()
+    end
```

Comment:
**question:** can we explicitly set `inserted_at` on these test pa messages and remove these calls to `Process.sleep/1` so this test doesn't have to take a fully 2s to run every time?